site/docs/recipes-api.md

```diff
@@ -112,3 +112,40 @@ export const button = recipe({
   }
 });
 ```
+
+## TypeScript
+
+Our API's are completely typed, meaning `recipe` return value will autocomplete as expected.
+
+If you want to use the type of your variants from a Component or other parts of your code, you can get that type using `RecipeVariants`:
+
+```ts
+// button.css.ts
+import { recipe } from '@vanilla-extract/recipes';
+import { reset } from './reset.css.ts';
+import { sprinkles } from './sprinkles.css.ts';
+
+export const button = recipe({
+  variants: {
+    color: {
+      neutral: sprinkles({ background: 'neutral' }),
```

Comment:
Think it's better to keep this example focussed on just the recipes API. Could we just use regular style blocks rather than sprinkles calls?

Reply:
Yep let me change this, makes sense!